addon/components/infinity-loader.js

```diff
@@ -68,14 +54,14 @@ export default Ember.Component.extend({
     if (Ember.typeOf(scrollable) === 'string') {
       var items = Ember.$(scrollable);
       if (items.length === 1) {
-        this.set('scrollable', items.eq(0));
+        this.set('_scrollable', items.eq(0));
       } else if (items.length > 1) {
         throw new Error("Multiple scrollable elements found for: " + scrollable);
       } else {
         throw new Error("No scrollable element found for: " + scrollable);
       }
-    } else {
-      this.set('scrollable', Ember.$(window));
+    } else if (scrollable === undefined || scrollable === null) {
+      this.set('_scrollable', Ember.$(window));
     }
```

Comment:
We should throw an error in the case when scrollable is neither a `string` or `null`.

I'm trying to prefix all errors with `"Ember Infinity: ..."` - could u update the two errors on like 73 & 75 also?